packages/moonstone/MoonstoneDecorator/fontGenerator.js

```diff
@@ -4,78 +4,78 @@
  * included by external developers.
  */
 // eslint-disable-next-line no-var
-var {addLocalizedFont, generateFontRules, generateFontOverrideRules} = require('@enact/ui/internal/localized-fonts');
+let {addLocalizedFont, generateFontRules, generateFontOverrideRules} = require('@enact/ui/internal/localized-fonts');
 
-const fontName = 'Moonstone Global';
+const fontName = 'Moonstone';
 
 // Locale Configuration Block
 const fonts = {
-	'NonLatin': {
-		// Hacky solution to add support for full-name and postscript names of the same font (support for multiple different operating systems font handling)
-		regular: 'LG Smart UI AR HE TH Regular"), local("LGSmartUIARHETH-Regular',
-		bold:    'LG Smart UI AR HE TH SemiBold"), local("LGSmartUIARHETH-SemiBold'
-	},
-	'am': {
-		regular: 'LG Smart UI Amharic'
-	},
-	'bn': {
-		regular: 'LG Smart UI Bengali'
-	},
-	'gu': {
-		regular: 'LG Smart UI Gujarati'
-	},
+	// 'NonLatin': {
```

Comment:
Can we remove these commented sections?

Reply:
Yes. I wasn't sure whether we wanted or needed them for later. I figured I'd save them just in case, but it's also completely fine to just remove them. I'll drop them now.